app/assets/stylesheets/markdown-content.scss

```diff
@@ -3,8 +3,9 @@
   p:last-child { margin-bottom: 0; }
 
   ul, ol {
-    margin-top:0.8em;
-    margin-bottom:0.8em;
+    margin-top: 0.8em;
```

Comment:
- `0.8` should be written without a leading zero as `.8`
- Properties should be ordered margin-bottom, margin-top